dist/tools/nacl_key_generator/README.md

```diff
@@ -0,0 +1,13 @@
+# Key-pair generator using [NaCL](https://nacl.cr.yp.to/)
+This program will generate two key-pairs and store them as binary files, using
+tweetNaCl package in RIOT
```

Comment:
I actually would prefer than a shell script that checks out the source from [1] instead of essentially duplicating the source.

[1] https://github.com/RIOT-OS/tweetnacl